doc/source/user_guide/index.rst

```diff
@@ -0,0 +1,30 @@
+{{ header }}
+
+.. _user_guide:
+
+==========
+User Guide
+==========
+
```

Comment:
A bit of introductory text here would be nice, explaining this section of the docs.

---

The User Guide covers all of pandas *by topic area*. Each of the subsections introduces
a topic (such as "working with missing data"), and discusses how pandas approaches the
problem, with many examples throughout.

Users brand-new to pandas should start with ... (10 min?)

Further information on any specific method can be obtained in the API reference.

---

We might also want to link to this page from a few places

- overview
- 10min
- dsintro
- basics

Reply:
I 100% agree, but it makes more sense to me to work on those in separate PRs after the whole restructure. There is already another PR for a top level "Getting started" section, that probably should be the linked for the "Users brand-new..." comment.

I think moving all the pages and finding an agreement on all the structures will actually take quite a lot of time, and will require a decent amount of discussion. That's why I'm proposing smaller PRs, to be able to keep some progress. Also, I think that will let us see the progress at every step, and make us move in the right direction (kind of agile methodology).

I mentioned it somewhere, but it was already work on trying to migrate all the documentation in a one-shot (see https://pandas-dev.github.io/pandas-sphinx-theme/pr-datapythonista_base/). After all the work on that, I think the approach will fail. If anyone else thinks a single PR or bigger PRs are better, I'm more than happy to leave this for someone else.